package.json

```diff
@@ -68,7 +68,7 @@
   },
   "dependencies": {
     "@ensdomains/ensjs": "3.0.0-alpha.57",
-    "@tableland/sdk": "4.0.2",
```

Comment:
should it be exact version to match what was there? ie, no `^` ?

Reply:
It is important for ENS to be pinned, the others it is not.

Reply:
Oh, the reason; ENS is in alpha and _will_ break things sometimes. But the pinned version cooperates.